levels/sl/areas/script_vanilla.inc.c

```diff
@@ -2,4 +2,4 @@ JUMP_LINK(script_func_global_1),
 JUMP_LINK(script_func_global_8),
 JUMP_LINK(script_func_global_17),
 JUMP_LINK(script_func_vo_sl),
-LOAD_MODEL_FROM_GEO(MODEL_SL_SNOW_TREE,          snow_tree_geo),
+LOAD_MODEL_FROM_GEO(MODEL_SNOW_TREE,          snow_tree_geo),
```

Comment:
Shouldn't this go in the actual vanilla objects file?

Reply:
Wait no it shouldn't, this is part of common1 (so it should be global then)

Reply:
Now that I'm looking at it...there's actually a lot of common1 geos unaccounted for here. I still think it makes the most sense to globally load all of them, but I guess technically this is a (mostly insignificant) waste of RAM for most hacks.